_script/setup

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2021 Adevinta
+
+set -e
```

Comment:
Not a big deal but I would add the "-u" flag in the `set` command above, just to stop the execution of the script before using a variable that is not bound.

Reply:
Fixed in commit b0feeb3b4855a3d1fd63168e568883d82acee034